manon/tsconfig.json

```diff
@@ -1,7 +1,14 @@
 {
-  "include": ["*.js", "**/*.js"],
   "compilerOptions": {
+    "module": "esnext",
+    "target": "esnext",
```

Comment:
Is `esnext` only used for typechecking or is it also used to compile/build the js?

If it is used for compiling is it safe to use esnext? / Is the browser support good enough?